services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -125,17 +125,22 @@ async def send_mail(app: web.Application, msg: MIMEText):
         hostname=cfg.SMTP_HOST,
         port=cfg.SMTP_PORT,
         use_tls=cfg.SMTP_TLS_ENABLED,
+        start_tls=cfg.SMTP_STARTTLS_ENABLED,
     )
     log.debug("Sending email with smtp configuration: %s", pformat(smtp_args))
     if cfg.SMTP_PORT == 587:
         # NOTE: aiosmtplib does not handle port 587 correctly
         # plaintext first, then use starttls
         # this is a workaround
         smtp = aiosmtplib.SMTP(**smtp_args)
-        await smtp.connect(use_tls=False, port=cfg.SMTP_PORT)
-        if cfg.SMTP_TLS_ENABLED:
-            log.info("Starting TLS ...")
+        if cfg.SMTP_STARTTLS_ENABLED:
+            log.info("Unencrypted connection attempt to mailserver ...")
```

Comment:
VERY MINOR: these kind of messages, I think are more debug messages than informative. I guess you used them to follow what is going on.